### ISSUE TITLE
Level selection images resize

### ISSUE THREAD

**Issue by phet-steele** [phetsims/fraction-matcher#89]
With increasing length of level title strings ("Level 1", "Level 2" <-- those things), the text scales down to fit the button. My gripe with this is that the image for each level also becomes smaller. They get waaay too small with ?stringTest=long. I see no need for this, there's still plenty of room for the images on the button to stay their normal size:

Before stringTest:
![fm dubs2](https://cloud.githubusercontent.com/assets/5863899/12492754/09c9768e-c03f-11e5-9a17-53567eb1c176.PNG)

After stringTest:
![fm dubs3](https://cloud.githubusercontent.com/assets/5863899/12492770/155449de-c03f-11e5-8ef1-2ba01a089c0e.PNG)

Tested on phettest with Win 8.1 FF. For phetsims/tasks/issues/479

Troubleshooting information:
Name: ‪Fraction Matcher‬:‪Fraction Matcher‬
URL: http://phettest.colorado.edu/fraction-matcher/fraction-matcher_en.html?ea&brand=phet&stringTest=double
Version: 1.0.0 (require.js)
Features missing: touch
User Agent: Mozilla/5.0 (Windows NT 6.3; WOW64; rv:43.0) Gecko/20100101 Firefox/43.0
Language: en-US
Window: 1366x633
Pixel Ratio: 1/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0
Vendor: Mozilla (Mozilla)
Vertex: attribs: 16 varying: 28 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

**Comment by phet-steele** [comment]
@samreid @ariel-phet this looks like it was overlooked for S2015R, can you share your comments?

**Comment by ariel-phet** [comment]
@phet-steele agreed, the text should scale, but the button image should not scale at all, assigning to @samreid

**Comment by samreid** [comment]
I'll work on this once RC testing is complete since this work can be done on the branch.

**Comment by samreid** [comment]
I've created a vegas issue for the root of the problem in https://github.com/phetsims/vegas/issues/49 and @ariel-phet recommended to publish S2015R without this fix.

**Comment by samreid** [comment]
I'll put this issue on hold until the vegas issue is closed, though be aware that was marked as low priority "to be revisited after all redeploys".

**Comment by samreid** [comment]
Unassigning, if/when the vegas issue is addressed, we can iterate over use cases (including this) and make sure they are good to go.

**Comment by ariel-phet** [comment]
This issue appears to be addressed, closing
